po/glossary/it.po

```diff
@@ -776,22 +775,20 @@ msgid "tax info"
 msgstr "info imposte"
 
 #. "Amost everybody has to declare and probably pay it. See https://en.wikipedia.org/wiki/Income_tax"
-#, fuzzy
-#| msgid "account type: Income"
 msgid "tax type: income tax"
-msgstr "tipo conto: entrate"
+msgstr "tipo tassa: imposta sul reddito"
```

Comment:
Minor:
The announcement in https://hosted.weblate.org/projects/gnucash/glossary/ says:
`In case a term contains a colon [:], translate only the part to the right of the colon. Do not translate terms in brackets [(…)] or behind commas[,]. They serve only for your better understanding of the meaning.`

Reply:
Ok, but there are plenty of other strings with colon that have been translated.

Reply:
There are plans to convert the part before the colon in msgstr to msgcontext.